index.html

```diff
@@ -1642,6 +1642,15 @@ <h2 id="OfflineAudioContext">
             </dl>
           </dd>
           <dt>
+            readonly attribute unsigned long length
```

Comment:
The type `unsigned long` matches the constructor, but it doesn't match `AudioBuffer.prototype.length`, which is just `long`. Filed https://github.com/WebAudio/web-audio-api/issues/747 for a similar issue.